db/migrate/20150107044854_users_without_location_cleanup.rb

```diff
@@ -0,0 +1,9 @@
+class UsersWithoutLocationCleanup < ActiveRecord::Migration
+  def up
+    User.delete_all(location: [nil,''])
```

Comment:
I need to check how many users have no location.  In any case users need to be destroyed instead of deleted. 
I prefer to set a dummy value instead of deleting/destroying Users.